administrator/components/com_media/controllers/file.json.php

```diff
@@ -56,6 +56,7 @@ public function upload()
 		// Get some data from the request
 		$file   = $this->input->files->get('files', '', 'array');
 		$folder = $this->input->get('folder', '', 'path');
+		$canOverwrite = $this->input->getBool('allowovewrite', false);
```

Comment:
@roland-d The other option for creating a copy is to have a flag (e.g. createNew) and deal with it in the controller logic (this one I think is a better/easier solution)